include/ppl/kernel/llm/cuda/pmx/vision_embedding.h

```diff
@@ -0,0 +1,64 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+#ifndef __PPL_KERNEL_LLM_CUDA_PMX_VISION_EMBEDDING_H__
+#define __PPL_KERNEL_LLM_CUDA_PMX_VISION_EMBEDDING_H__
+
+#include <cudnn.h>
+
+#include "ppl/kernel/llm/cuda/common/general_include.h"
+
+namespace ppl { namespace kernel { namespace llm { namespace cuda { namespace pmx {
+
+struct vision_embedding_config {
+    int32_t hidden_dim;
+    int32_t batch_size;
+    int32_t image_channel;
+    int32_t image_size;
```

Comment:
Consider `image_h` and `image_w`?
We usage both height and width in python implement: https://github.com/openppl-public/ppl.pmx/blob/master/torch_function/VisionEmbedding.py#L23